windows/RNSVG/RenderableView.cpp

```diff
@@ -358,20 +364,29 @@ void RenderableView::Unload() {
 
 RNSVG::IRenderable RenderableView::HitTest(Point const &point) {
   if (m_geometry) {
-    bool strokeContainsPoint{false};
+    BOOL strokeContainsPoint{false};
```

Comment:
I guess technically both of these `BOOL` types should init to `FALSE` instead of `false`, but 🤢